pkg/setting/setting.go

```diff
@@ -88,6 +88,7 @@ var (
 	CookieSameSiteMode     http.SameSite
 
 	// Snapshots
+	SnapshotEnabled       bool
```

Comment:
we don't [encourage introducing new global settings](https://github.com/grafana/grafana/blob/main/contribute/backend/README.md#settings-refactoring): we prefer to have them `setting.Cfg` instead.

Reply:
Thanks for the review @papagian !
I move the new config, as well as the old one to seeting.Cfg here d815db36c21fc61645c4ff15063aaceea78798e6
Also, just FYI, I did some minor changes to fix some warning for some variable names b0d60921101ba4b39e8c57671e7aa7d11ea2d7c2